app/src/main/java/com/itssuryansh/taaveez/Edit_Content.kt

```diff
@@ -46,14 +56,24 @@ class Edit_Content : AppCompatActivity() {
         loadDayNight()
         id = intent.getIntExtra(Constants.ID,0)
         super.onCreate(savedInstanceState)
+
+        // Selected Labels
+        selectedLabels = HashSet()
+
+        // Label Suggestions
+        labelSuggestions = mutableListOf()
```

Comment:
I know here you've done like this just for sample, would be better if they're saved in a separate Table, The note table would only have the id's of the labels and not the label as string